graph-db/serverless/src/indexCmrCollection/__test__/handler.test.js

```diff
@@ -0,0 +1,22 @@
+const { indexCmrCollection } = require('../handler')
+
+jest.mock('../../../../commonUtils/cmr/getEchoToken')
+jest.mock('../../../../commonUtils/cmr/fetchCmrCollection')
+jest.mock('../../../../commonUtils/gremlin/initializeGremlinConnection')
+jest.mock('../../../../commonUtils/indexing/indexCmrCollection')
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
+
+describe('indexCmrCollection handler', () => {
+  test('test index of single collection', async () => {
+    const event = { Records: [{ body: '{"concept-id": "C1237293909-SCIOPS" }' }] }
+
+    const indexed = await indexCmrCollection(event)
+
+    const { body, statusCode } = indexed
+    expect(body).toBe('Collection [C1237293909-SCIOPS] indexed sucessfully: true')
```

Comment:
It would be nice to verify that the collection vertex actually exists in graph db, rather than the response says thing are indexed. The code could not work at all and returns a response that says things are indexed successfully.

Reply:
similar comments on related url vertex and the `documents` edge. We should verify they exists in the graph db.